internal/entities/handlers/strategies/entity/add_originating_entity.go

```diff
@@ -108,10 +123,9 @@ func (a *addOriginatingEntityStrategy) GetEntity(
 		}
 
 		// Persist the properties
-		upstreamProps, err = a.propSvc.RetrieveAllProperties(ctx, prov,
-			parentEwp.Entity.ProjectID, parentEwp.Entity.ProviderID,
-			childProps, entMsg.Entity.Type,
-			propertyService.ReadBuilder().WithStoreOrTransaction(t),
+		err = a.propSvc.SaveAllProperties(ctx, entID,
```

Comment:
not related to this change, but to earlier calling `FetchAllProperties` - this means that the message needs different properties depending on which handler we call. If we call the `TopicQueueRefreshEntityAndEvaluate` we pretty much need an upstream ID. If we call `TopicQueueOriginatingEntityAdd` the message must contain properties that are passed to the upstream call, in github's case repo owner, repo name and ID. I don't have a great idea on how to solve this except comments and clear error messages.

I just hate when I try to design an API and the result turns out this way..